src/components/Home.tsx

```diff
@@ -13,8 +12,19 @@ import ConnectWalletScreens from './MultiModal/ConnectWallet/ConnectWalletScreen
 import Loader from './Loader'
 import Onboarding from './Onboarding'
 
+const DynamicSection = styled.div<{
+  marginTop: any
+}>`
+  margin-top: ${props => props.marginTop}px;
+  padding: 0 ${2 * GU}px;
+`
+
+const LoadingSection = styled.div`
+  margin: ${5 * GU}px 0;
+`
```

Comment:
I think we already defined some styled-components this way and we decided to put them at the bottom after the main logic. What do you think?

Reply:
In my company we have add them to a separate file like header.styled.ts, and create a folder called Home and put the Header.tsx and the styled.ts inside it.

Reply:
Interesting, we can ask the rest of the team how they feel with the different options and stick to one of them going forward 👍

Reply:
Yah i'm used to make each style that way. Component and styled.ts separated, its get alot the folder but it get better refactor and less size by file.

Reply:
What I make different is use type or interface to props. So it can be:


```typescript
interface IDynamicSectionProps {
  marginTop: any
};

const DynamicSection = styled.div<IDynamicSectionProps>`
margin-top: ${props => props.marginTop}px;
padding: 0 ${2 * GU}px;
`

```

Reply:
@kamikazebr Yeah that's the same thing. But its the best way for sure 🔥